BranchMonsterFactory/ViewControllers/MonsterCreatorViewController.m

```diff
@@ -51,9 +53,18 @@ - (void)viewDidLoad {
         [currView.layer setCornerRadius:currView.frame.size.width/2];
     }
     
+    //if it's null, create a new default monster
+    if (!self.editingMonster) {
+        self.editingMonster = [[BranchUniversalObject alloc] initWithCanonicalIdentifier:[NSString stringWithFormat:@"MONSTER_%u", arc4random_uniform(10000)]];
```

Comment:
instead of a random number, can we use the face index, the body index, the color index, and the monster name (and any other important variables)? that way it's truly a canonical identifier.